src/assets/scss/component/_custom-form-styles.scss

```diff
@@ -58,502 +55,494 @@ $formContainerBorderColor: #ccc;
+    height: 32px;
+    vertical-align: middle;
+  }
+  .ss-grid {
+    overflow-x: auto;
+  }
+  .ss-gridnumbers {
+    text-align: center;
+    border-bottom: 1px solid $gridNumbersBorderColor;
+  }
+  .ss-gridnumber {
+    display: block;
+    padding: 0.5em 0;
+  }
+  .ss-gridrow {
+    text-align: center;
+    color: $baseColor;
+    border-bottom: 1px solid $gridRowBorderColor;
+    padding: 0.5em 0.25em;
+  }
+  .ss-grid-row-even {
+    background-color: $gridRowEvenBackground;
+  }
+  .ss-grid-row-odd {
+    background-color: $gridRowOddBackground;
+  }
+  .ss-gridrow-leftlabel {
+    min-width: 100px;
+    max-width: 200px;
+    padding-left: 15px;
+    text-align: left;
+  }
+  .ss-grideditor-columns input {
+    line-height: 150%;
+  }
+  .ss-grideditor-editor .ss-magiclist-ul span.ss-header {
+    padding-right: 1em;
+  }
+  .ss-grid-separator {
+    border: 1px solid $gridSeparatorBorderColor;
+  }
+  .ss-scalenumbers {
+    text-align: center;
+  }
+  .ss-scalenumber {
+    display: block;
+    padding: 0.5em 0;
+  }
+  .ss-scalerow {
+    text-align: center;
+    color: $baseColor;
+    border: 1px solid $scaleRowBorderColor;
+    border-left: 0;
+    border-right: 0;
+    padding: 0.5em 0.25em;
+  }
+  td.ss-leftlabel {
+    text-align: right;
+    padding-left: 0;
+  }
+  td.ss-rightlabel {
+    text-align: left;
+    padding-right: 0;
+  }
+  .ss-form-entry {
+    margin-bottom: 1.5em;
+  }
+  .ss-form-entry input {
+    vertical-align: middle;
+    margin-left: 0;
+    margin-top: 0;
+  }
+  .ss-form-entry select {
+    max-width: 100%;
+  }
+  .ss-q-title {
+    display: block;
+  }
+  .ss-q-help {
+    display: block;
+    margin: 0.1em 0 1rem;
+  }
+  .ss-secondary-text {
+    color: $secondaryColor;
+    font-size: $bodyFontSize;
+    margin-top: -.5em;
+  }
+  .ss-textarea {
+    max-width: 99%;
+  }
+  .ss-q-long {
+    resize: vertical;
+    width: 70%;
+  }
+  .ss-choices {
+    list-style: none;
+    margin: 0.5em 0 0;
+    padding: 0;
+  }
+  .ss-choice-item {
+    margin: 0;
+    line-height: 1.3em;
+    padding-bottom: 0.5em;
+    display: $listChoiceItem;
+  }
+  .ss-required-asterisk {
+    color: $error;
+  }
+  .required-message {
+    display: none;
+  }
+  .ss-section-title {
+    background-color: $sectionTitleBackground;
+    padding: 0.4em;
+    margin: 2em -0.4em 0;
+  }
+  .ss-section-description {
+    margin-top: 0.5em;
+  }
+  .ss-page-title {
+    margin: 0;
+    padding: 0;
+  }
+  .ss-formwidget-container:focus {
+    outline: none;
+  }
+  .ss-item {
+    margin: $bodyLineHeight*1em 0;
+  }
+  .ss-image,
+  .ss-image-container {
+    max-width: 100%;
+  }
+  .ss-image:focus {
+    outline: none;
+  }
+  .img-menu .jfk-button-collapse-left {
+    margin-right: 0;
+  }
+  .ss-edit-link,
+  .ss-edit-link:active,
+  .ss-edit-link:visited,
+  .ss-edit-link:hover,
+  .ss-edit-link:link {
+    background: url("//www.gstatic.com/docs/spreadsheets/forms/edit_dark.png") 7px 8px no-repeat #1f1f1d;
```

Comment:
Color literals like `#1f1f1d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>URL `//www.gstatic.com/docs/spreadsheets/forms/edit_dark.png` should not contain protocol or domain